src/masonite/authentication/models/authenticates.py

```diff
@@ -18,19 +17,15 @@ def attempt(self, username, password):
         record_password = getattr(record, self.get_password_column())
         if not isinstance(record_password, bytes):
             record_password = bytes(record_password or "", "utf-8")
-        if bcrypt.checkpw(bytes(password, "utf-8"), record_password):
+        if Hash.check(password, record_password):
```

Comment:
Yup this is exactly how the facades should be used. We no longer have to import the wsgi file